main/integration-tests/__test__/api-tests/common/workspace-service-catalogs/connections/send-ssh-public-key-connection.test.js

```diff
@@ -37,6 +40,11 @@ describe('Send SSH public key scenarios', () => {
 
   afterAll(async () => {
     await deleteDefaultServiceCatalogProduct(setup, productInfo);
+    await Promise.all(
+      _.map(dummyWorkspacesToDelete, async envId => {
```

Comment:
Since `dummyWorkspacesToDelete` is an array, I don't think you need to use lodash. You can just do 
```
dummyWorkspacesToDelete.map(async(workspace) => {
    return deleteWorkspaceServiceCatalog({ aws: setup.aws, id: envId });
});
```

Reply:
True, but I try to stick with lodash as a pattern due to its [performance](https://measurethat.net/Benchmarks/Show/2676/0/arrayprototypemap-vs-lodashmap#latest_results_block)